gremlin-core/src/main/java/org/apache/tinkerpop/gremlin/process/traversal/dsl/graph/GraphTraversal.java

```diff
@@ -4078,6 +4092,7 @@ private Symbols() {
         public static final String dateDiff = "dateDiff";
         public static final String all = "all";
         public static final String any = "any";
+        public static final String listNone = "none";
```

Comment:
Could you have just called dthis none instead of listNone?

Reply:
The "none" token is already on `Traversal.Symbols` - shouldn't that just be used?